.github/workflows/ci.yml

```diff
@@ -245,7 +245,7 @@ jobs:
 
       - name: Trufflehog secrets scanning
         if: ${{ inputs.run-trufflehog == true }}
-        uses: grafana/plugin-ci-workflows/actions/plugins/trufflehog@main
+        uses: grafana/plugin-ci-workflows/actions/plugins/trufflehog@giuseppe/trufflehog-only-verified
```

Comment:
I guess this should be reverted (or can it only be done once merged first? 😵)

Reply:
Ah, yes :sweat_smile: I have reverted this change